akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1934,6 +1935,8 @@ trait FlowOpsMat[+Out, +Mat] extends FlowOps[Out, Mat] {
   def watchTermination[Mat2]()(matF: (Mat, Future[Done]) ⇒ Mat2): ReprMat[Out, Mat2] =
     viaMat(GraphStages.terminationWatcher)(matF)
 
+  def monitor: ReprMat[Out, (Mat, FlowMonitor[Out])] = viaMat(Flow.monitor[Out])(Keep.both)
```

Comment:
I tried putting it in FlowOps, but then I couldn't change the materialized value.
Am I missing something?